lib/howl/buffer.moon

```diff
@@ -277,6 +278,19 @@ class Buffer extends PropertyObject
       b_end_pos = @byte_offset end_pos
       @_buffer\ensure_styled_to pos: b_end_pos
 
+  mode_at: (pos) =>
```

Comment:
Regarding these two `{config,mode}_at`, I see that some code has been updated to use them, and others not. Is there a reason for that?